conformance/packages/dns-test/src/client.rs

```diff
@@ -117,10 +124,12 @@ impl Client {
 }
 
 #[derive(Clone, Copy)]
-pub struct DigSettings {
+pub struct DigSettings<'a> {
```

Comment:
Adding the lifetime here is a bit yucky, but the `Copy` derive limits what can be done for a setting that needs a string argument. If we use an owned `String` then we have to drop `Copy` and there's _so many_ instances of this type being copied that would need updating...